### DISCUSSION TITLE
Volume mount permissions using Docker Compose

### DISCUSSION THREAD

**Discussion by ralf-berger** [ude-soco/RIMA#344] [Q&A]
A permission issue was brought up [in an unrelated issue](https://github.com/ude-soco/RIMA/issues/332#issuecomment-1691043269):

`rima-model-downloader-1  | mkdir: cannot create directory ‘/home/app/.model/nltk’: Permission denied`

Meaning that the directory `./tmp/model` from the working copy, mounted as `/home/app/.model` in the `model-downloader` container, is not writable.

@shoebjoarder Did you make any progress [checking your user and permissions](https://github.com/ude-soco/RIMA/issues/332#issuecomment-1692849750)?

**Comment by shoebjoarder** [comment]
I made progress, I altered it to writable. The build was successful, and the frontend is running. However, I am facing CORS issue when sending request to /register
